README.md

```diff
@@ -146,6 +146,14 @@ Autoload can be customised using the following options:
     forceESM: true
   })
   ```
+- `encapsulate` (optional) - Defaults to 'true', if set to 'false' each plugin loaded is wrapped with [fastify-plugin](https://github.com/fastify/fastify-plugin). This allows you to share contexts between plugins and the parent context if needed. For example, if you need to share decorators. Read [this](https://github.com/fastify/fastify/blob/main/docs/Reference/Encapsulation.md#sharing-between-contexts) for more details.
```

Comment:
Would this option be like all the plugins are in the same context?
For example this directory tree:

```
/foo
 - foo.js
/bar
 - bar.js
 - baz.js
```

All the *.js files are loaded in the same context?

Would not be more useful to create 2 contexts in this case: one for the `foo/` dir and the `bar/` one?